BGAnimations/ScreenSystemLayer overlay.lua

```diff
@@ -210,4 +210,57 @@ t[#t+1] = LoadFont("Common Footer")..{
 	end
 }
 
+-- -----------------------------------------------------------------------
+-- Modules
+
+local function LoadModules()
+	-- A table that contains a [ScreenName] -> Table of Actors mapping.
+	-- Each entry will then be converted to an ActorFrame with the actors as children.
+	local modules = {}
+	local files = FILEMAN:GetDirListing(THEME:GetCurrentThemeDirectory().."Modules/")
+	for file in ivalues(files) do
+		-- Get the file extension (everything past the last period).
+		local filetype = file:match("[^.]+$"):lower()
+		if filetype == "lua" then
+			local full_path = THEME:GetCurrentThemeDirectory().."Modules/"..file
+			Trace("Loading module: "..full_path)
+
+			-- Load the Lua file as proper lua.
+			local loaded_t = dofile(full_path)

```

Comment:
We should probably protect the theme from errors in a module, so that adding a broken module doesn't break it. We could use `pcall` for that.

Reply:
Turns out loadfile does this as well so I ended up just making use of that (returns the chunk as a function, or return nil + error message).

Excerpt from the log can be found below:

00:00.810: Loading module: Themes/Simply-Love-SM5-mainline/Modules/print.lua
00:00.811: Error loading module: Themes/Simply-Love-SM5-mainline/Modules/print.lua with error:
00:00.811:     Themes/Simply-Love-SM5-mainline/Modules/print.lua:3: '=' expected near 'is'